### DISCUSSION TITLE
ClickHouse versions

### DISCUSSION THREAD

**Discussion by bpetitcollot** [plausible/analytics#3026] [Self-Hosted Support]
Hello,

I am self-hosting plausible based on [plausible/hosting](https://github.com/plausible/hosting) template.

I recently faced up [an issue](https://github.com/ClickHouse/ClickHouse/issues/50885) with the backup/restore feature of ClickHouse which was "solved" with upgrading the ClickHouse version.

As plausible docker-compose rely on v.22.6 which is no longer supported, and considering the fast release frequency of ClickHouse (LTS are supported for only one year after release), i would like to know **how Plausible deals with ClickHouse versions**.

**Comment by ruslandoga** [comment]
👋 @bpetitcollot 

AFAIK Plausible is using `22.8.13.20` in prod. [And it's also the version that's being tested against.](https://github.com/plausible/analytics/blob/a356009af8cbdbf3c37670c9c9566cf76e640f4e/.github/workflows/elixir.yml#L32)

**Reply by ruslandoga** [reply]
It seems like updating `plausible/hosting` to use `22.8.13.20` as well is a good idea.

**Reply by ruslandoga** [reply]
PR: https://github.com/plausible/hosting/pull/72
